docs/api/saved-objects/export_by_type.asciidoc

```diff
@@ -0,0 +1,28 @@
+[[saved-objects-api-export-by-type]]
+=== Export Objects By Type
+
+experimental[This functionality is *experimental* and may be changed or removed completely in a future release.]
+
+The export saved objects by type API enables you to retrieve a set of Kibana saved objects into a newline delimited JSON format.
+
+Note: You cannot access this endpoint via the Console in Kibana.
+
+==== Request
+
+`POST /api/saved_objects/_export_by_type/{type}`
+
+==== Response body
+
+The response body will have a format of newline delimited JSON. The order of these objects is not guaranteed other than having dependencies defined first.
```

Comment:
I thought we had planned on sorting the objects when we request them from Elasticsearch so that then they are sorted with dependencies first they would always be in the same order?